src/protocols/ethereum/components/TransferReview.vue

```diff
@@ -66,6 +67,10 @@ export default defineComponent({
 
     const loading = ref<boolean>(false);
 
+    const isSelectedAssetErc20 = computed(
+      () => props.transferData?.selectedAsset?.contractId !== ETH_CONTRACT_ID,
```

Comment:
What do you think about using `getCoinContractId` for obtaining the coin contract ID? I personally feel that the adapter should be the most reliable source of truth.